### PR TITLE
Localizing controllers

### PR DESCRIPTION
**Pull request by taharushain**
- till Marketplace Controller

### CODE REVIEW

File: app/controllers/marketplaces_controller.rb [hunk 1]
```diff
@@ -110,24 +110,24 @@ def update_account
       account = @marketplace.update_account
       if account
         if account.id
-          @activation_message = "Conseguimos com sucesso atualizar sua conta do #{@marketplace.organizer.name}"
+          @activation_message = t("marketplace_controller_activation_message_five")
```
Comment (alexanmtz) [inline app/controllers/marketplaces_controller.rb]: Here will be:

`t('marketplace_controller_activation_message_five', organizer: @marketplace.organizer.name) %>`

And on yaml language file file:
`marketplace_controller_activation_message_five: "Conseguimos com sucesso atualizar sua conta do %{organizer}"`

when has variable there's a second parameter in the t function, and on the string you mark with `%{variable}`

So this needs to be fixed in all translations that holds a variable

**Comment by alexanmtz** [comment]
[Gerenciador de textos e idiomas](https://trello.com/c/6A2uNPkf/139-gerenciador-de-textos-e-idiomas)

**Comment by taharushain** [comment]
Please fix the tests, so that it could be merged.

Thanks.

**Comment by taharushain** [comment]
Hey Alexandre, 

All the suggested views have been converted to i18n, please perform the necessary steps to take this kid into production.

**Pull request merged**